web-src/src/components/ContainerView.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import React, { useEffect, useState } from 'react';
+import { Content, Grid, Heading, IllustratedMessage, View } from '@adobe/react-spectrum';
+import WriteIcon from '../icons/WriteIcon';
+import Editor from './Editor';
+
+function ContainerView() {
+
+  return (
+    <Grid
+      areas={[
+        'prompt variations',
+      ]}
+      columns={['1fr', '1fr']}
+      rows={['auto']}
+      height="100vh"
+      >
+      <View gridArea="prompt" overflow="auto" height="90vh" UNSAFE_style={{"paddingRight": "30px"}}>
```

Comment:
Would it be alright to use viewport width and height in nested elements?
Could it be easier to use the size of the parent container, especially if we need to move things around?
/cc @askayastha22